samples/algorithms/sudoku-grover/Program.cs

```diff
@@ -0,0 +1,474 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Microsoft.Quantum.Samples.SudokuGrover
+{
+    class Program
+    {
+        /// # Summary
+        /// Main entry point
+        ///
+        /// # Inputs
+        /// ## args
+        /// add the following argument to specify which puzzles to run
+        /// -  `all` or blank : run all puzzles (default)
+        /// -  `4x4-classic` : test classic algorthm on a 4x4 puzzle
+        /// -  `4x4-1` : test Quantum solution of 4x4 puzzle missing 1 number
+        /// -  `4x4-3` : test Quantum solution of 4x4 puzzle missing 3 numbers
+        /// -  `4x4-4` : test Quantum solution of 4x4 puzzle missing 4 numbers
+        /// -  `9x9-1` : test classic algorithm and Quantum solution on a 
+        ///              9x9 puzzle with 1 missing number
+        /// -  `9x9-2` : test Quantum solution on a 
+        ///              9x9 puzzle with 2 missing numbers
+        /// -  `9x9-64` : test classic algorithm and Quantum solution on a 
+        ///              9x9 puzzle with 64 missing numbers
```

Comment:
C# uses XML instead of Markdown to format documentation comments, such that this will appear incorrectly in intelliSense as currently written.

Reply:
Docs here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/